static/app/components/workflowEngine/gridCell/connectionCell.tsx

```diff
@@ -6,69 +6,77 @@ import {Hovercard} from 'sentry/components/hovercard';
 import ProjectBadge from 'sentry/components/idBadge/projectBadge';
 import Link from 'sentry/components/links/link';
 import {EmptyCell} from 'sentry/components/workflowEngine/gridCell/emptyCell';
+import {tn} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
-import type {AvatarProject} from 'sentry/types/project';
-
-export type Item = {
-  link: string;
-  name: string;
-  project: AvatarProject;
-  description?: string;
-};
 
 export type ConnectionCellProps = {
-  items: Item[];
-  renderText: (count: number) => string;
+  ids: string[];
+  type: 'detector' | 'workflow';
```

Comment:
🤔 should the detector or workflow stuff exist on a context so we can easily grab that information in any of these components?

Reply:
I went down this road, but as soon as I started writing tests I realized this was an extra layer of indirection. The connection cells are flipped, so if the table is for detectors, the connection cell needs to fetch workflows and vice versa. Better to have this be an explicit prop to make it scannable, IMO.